_data/wdelenclos/wdelenclos.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Wladimir Delenclos
+institution: HETIC
+profile_pic: wdelenclos.png
+quote: I’ve learned from the bad times and was humbled by the good. Thank you for all of the great life lessons.
```

Comment:
Character count exceeds the limit of 100, you might want to modify this part

Reply:
Fixed, thanks